### DISCUSSION TITLE
不知道算不算bug 开机自启动时 无法执行相对路径 ../xxx.exe

### DISCUSSION THREAD

**Discussion by bytimer** [shuax/MouseInc.Settings#416] [Q&A] [answered]
直接以实例来说明，CLaunch和MouseInc两个软件的绝对路径如下
C:\Tools\MouseInc\MouseInc.exe
C:\Tools\CLaunch\CLaunch.exe
理论上在MouseInc中应该能用相对路径执行到 **..\CLaunch\CLaunch.exe**

全局热键和通用动作中,若使用相对路径   **Execute,  ..\CLaunch\CLaunch.exe**
则无论是以计划任务 还是在注册表HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run中添加的开机启动项
开机自启动MouseInc后，从MouseInc中都无法执行**..\CLaunch\CLaunch.exe** ，感觉是找不到路径
而改成绝对路径**C:\Tools\CLaunch\CLaunch.exe** 或 环境变量路径**%tools%\CLaunch\CLaunch.exe** （%tools% =C:\Tools\ ） 又都能执行到

另外，把开机自启动的MouseInc进程**退出**后，再**手动双击启动的MouseInc**，却能正常启动  **..\CLaunch\CLaunch.exe**
把MouseInc快捷方式放到开始-程序-启动文件夹，以此方式开机自启动的MouseInc 也能正常启动  **..\CLaunch\CLaunch.exe**

不知道是windows系统的bug，还是MouseInc的问题

**Comment by shuax** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/6940583/190984572-6efad3bd-3903-4eec-b444-e05d39f604ff.png)
最好通过`%appdir% `使用
